identity-and-access-management/secret-manager/api-cli/generate-secret-password.mdx

```diff
@@ -80,7 +80,7 @@ func main() {
     scw.WithDefaultOrganizationID("SCW_DEFAULT_ORGANIZATION_ID"),
     // Get your credentials at https://console.scaleway.com/iam/api-keys
     scw.WithAuth("SCW_ACCESS_KEY", "SCW_SECRET_KEY"),
-    // Get more about our availability zones at https://www.scaleway.com/en/docs/console/my-account/reference-content/products-availability/
+    // Get more about our availability zones at https://www.scaleway.com/docs/console/my-account/reference-content/products-availability/
```

Comment:
I could be wrong, but I thought https://www.scaleway.com/en/docs/etc was the true address, and https://www.scaleway.com/docs/ is just redirected to the former - and that therefore we "should" use the full /en address as preference when we link to docs?